contracts/interfaces/handlers/IBosonAccountHandler.sol

```diff
@@ -118,6 +121,24 @@ interface IBosonAccountHandler is IBosonAccountEvents {
      */
     function updateSeller(BosonTypes.Seller memory _seller, BosonTypes.AuthToken calldata _authToken) external;
 
+    /**
+     * @notice Opt-in to a pending seller update
+     *
+     * Emits a SellerUpdateApplied event if successful.
+     *
+     * Reverts if:
+     * - The sellers region of protocol is paused
+     * - Addresses are not unique to this seller
+     * - Caller is not the address pending update for the field being updated
+     * - Caller is not the address of the owner of the pending AuthToken being updated
```

Comment:
I think these revert reasons can be made clearer

- Caller address is not pending for the field being updated
- Caller is not the of the owner of the pending AuthToken being updated